package.json

```diff
@@ -8,7 +8,7 @@
     "build-ts": "tsc",
     "build": "npm run build-sass && npm run build-ts && npm run copy-assets && npm run minfiy-build-js",
     "build-dev": "npm run build-sass && npm run build-ts && npm run copy-assets",
-    "build-sass": "rm -rf dist/public/style.css && sass --load-path=node_modules/govuk-frontend/govuk --no-source-map src/assets/scss/application.scss dist/public/style.css --style compressed",
+    "build-sass": "rm -rf dist/public/style.css dist/public/language-toggle.css  && sass --load-path=node_modules/govuk-frontend/govuk --no-source-map src/assets/scss/application.scss:dist/public/style.css node_modules/@govuk-one-login/one-login-language-toggle/stylesheet/styles.css:dist/public/language-toggle.css --style compressed",
```

Comment:
Tell me if I'm wrong but you're using a new css file called language-toggle.css but don't use it in any file. Where is it imported?

Reply:
omg yes, good catch thank you !  I didn't realise that I hadn't added our styling because the styling from their previous implementation was being applied.

Reply:
I have added link tag to language-toggle.css in the base file